command/server/server.go

```diff
@@ -45,6 +50,11 @@ func NewServer(config *Config) (*Server, error) {
 		return nil, err
 	}
 
+	// start the Prometheus Server
+	if err := srv.StartPrometheus(config.Prometheus.address, config.Prometheus.Enabled); err != nil {
```

Comment:
Do not use a different function here, we already have one called setupTelemetry to perform telemetry setup.